openedx/features/colaraz_features/models.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
```

Comment:
As we don't have any models, for now, you can delete this file.

Reply:
If we consider this argument we might have to delete all files other than (__int__, app and middleware). If its recommended we should do this.
@tasawernawaz ^^

Reply:
remove them for now, we will create these when needed

Reply:
done

Reply:
Thanks Hamza

Reply:
Thanks @HamzaIbnFarooq